server/controllers/admin/employees.js

```diff
@@ -190,43 +216,43 @@ exports.update = function update(req, res, next) {
     employee.date_embauche = new Date(employee.date_embauche);
   }
 
-  db.exec(sql, [employee, req.params.id])
-  .then(function (row) {
+  delete(employee.debitor_group_uuid);
```

Comment:
This is a lot of `delete`s.  Could you instead redefined a new object, ignoring all other properties?  For example:
```
var clean = {
  date_embauche : employee.date_embauche,
  // all other properties you want to keep on the employee object
};

transaction.addQuery(sql, [clean]);
```